package exception;

public class Ex02 {

	public static void main(String[] args) {
		
		try {
		System.out.println(1 / 0);
		}catch(ArithmeticException ae){
			System.out.println("0으로 나눌수 없습니다.");
		}
		
	}

}
